packages/cozy-stack-client/src/AppCollection.js

```diff
@@ -0,0 +1,36 @@
+import { uri } from './utils'
+import { normalizeDoc } from './DocumentCollection'
+
+/**
+ * Abstracts a collection of documents of the same doctype, providing
+ * CRUD methods and other helpers.
+ */
+export default class AppCollection {
+  constructor(doctype, client) {
+    this.doctype = doctype
```

Comment:
The doctype for such a collection is always `io.cozy.apps`, it must be simplified with a optional and default valued parameter or without the parameter `doctype` at all.

Reply:
I choose to remove doctype parameter.